### DISCUSSION TITLE
BlockRank strategic partnership proposal

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#11] [General]
@AlgoRhythMatic and I had got a meeting with [BlockRank](https://block-rank.com/) a few weeks ago, got familiarized with their service, here's AlgoRhythMatic's report from that meeting:

-----

- Ipaleka and AlgoRhythMatic met with Davis Weaver from Blockrank (https://block-rank.com) via Zoom call
- Ipaleka described a bit of about ASA Stats background, purpose, and mission to reach DAO.
- AlgoRhythMatic described a bit about ASA Stats community approach to development and rewards, as well as his own experience w/ ASA Stats.
- Davis preceded to explain about BlockRank's goals and purpose, and provided an onscreen demonstration that displayed BlockRank's visual analysis of wallets holding and exchanging the BirdBot token - identifying several that appeared to be used for gaming "fairdrops".
- Davis and Ipaleka discussed potential for BlockRank to provide an API for use within ASA Stats widget-based approach.
- All parties identified a mutual interest in staying connected and reviewing possibility for API integrations in the future, once this part of the ASA Stats roadmap was in closer range.

Summary: a nice productive meet-and-greet! ASA Stats has made an informal connection with Davis from BlockRank, and had the opportunity to communicate what we are looking for in terms of a future API integration for a widget down the road.

-----

And this is my very recent email communication with Davis:

-----
Hi Ipaleka,

I wanted to reach out to you with a proposal for a strategic partnership between ASAstats and BlockRank.  We would love to partner with you all to help with your token distributions (i.e. airdrops) and collaborate on potential API endpoints you may be interested in. I've included a potential partnership proposal below. 

BlockRank provides:

One year service of the following: 

1. Any API endpoints that we build that you are interested in
2. Application to make sending the airdrops automated and user-friendly (launching early May)
3. Application to facilitate identification and removal of airdrop farmers from any and all token distribution (already live)
4. Dedicated customer support to make sure 2 and 3 are able to be used by the ASAstats team. We are even willing to send over lists of airdrop farmers that we identify using the software if your team has trouble with the learning curve.  

We are only forming between 5-10 strategic partners. We are in a product development cycle and really want input from folks we trust to help us add features that people will find useful. The benefit for you is - if you have something you need from an analytics standpoint, we'll add it to our backlog and likely build it for you!

ASAstats provides:

6000 USD equivalents of the ASAstats token (1-year vesting schedule) at the current ASAstats market price.

Joint promotion of one another's projects is a given as well! 

Looking forward to hearing your thoughts. 

Davis

-----

Hi Davis!

Thank you for your email and consideration. Based on the ASA Stats' base principle, we have to discuss every discussion with the community.

Is there some information from this proposal that you don't want to be publicly exposed?

Sincerely,

Ivica

-----

Not at all! Please share with your community.

-----

I will, thank you!

-----

So I'm opening a discussion about their proposal.

**Comment by AlgoRhythMatic** [comment]
> 1. Any API endpoints that we build that you are interested in 

I like the idea of a partnership here, and think that BlockRank is an interesting example of some functionality we could potentially use as a proof-of-concept for API widget.

> 2. Application to make sending the airdrops automated and user-friendly (launching early May)

Not so sure we need their automated airdrop functionality vs. our own, but would be interested to hear technical POV about pros/cons vs. what we use today.

> 3. Application to facilitate identification and removal of airdrop farmers from any and all token distribution (already live)
 
Also interesting...but maybe as a service thought API vs. for our own airdrops, as I am not sure that gaming our rewards is an acute concern based on weekly payout remaining very consistent.

> 4. Dedicated customer support to make sure 2 and 3 are able to be used by the ASAstats team. We are even willing to send over lists of airdrop farmers that we identify using the software if your team has trouble with the learning curve.

Summary: if the 6000k USD of ASASTATS is primarily meant for 2/3/4 - probably not worth it.  If it is meant to secure API licensing/integrations...could be worth it, but I think we'd need to understand the use case/s where this widget is then considered valuable to our end users.  I think the most ideal partnership we are looking for is one where ASASTATS and BlockRank provide equal value to each other - I think the proposals above are all primarily ones where BlockRank is purely selling us a service.

**Reply by ipaleka** [reply]
> Not so sure we need their automated airdrop functionality vs. our own, but would be interested to hear technical POV about pros/cons vs. what we use today.

We've invested in our own solution and for abandoning it we should have plenty of good reasons, like a pretty solid assurance for enough exposure of that *joint promotion*.

**Reply by ipaleka** [reply]
> Also interesting...but maybe as a service thought API vs. for our own airdrops, as I am not sure that gaming our rewards is an acute concern based on weekly payout remaining very consistent.

Afaic this one is a major obstacle for partnership based on the proposed points. 

The approach from the very start in ASA Stats has been very simple: if you are qualified for rewards and airdrops then your obligations ends at the moment you receive ASASTATS. The matter of fact is that we simply don't have an institute to prevent sending rewards and airdrops to anybody, no matter if they for certain are those *airdrop farmers*. Our only condition has been - and will remain in the pre-DAO phase - that they are qualified for a reward/airdrop.

**Comment by dragmz** [comment]
A fair partnership offer here is asastats.com introduces a widget for their service (so users interested in the service can pay for it) and they provide us with their API. Anyway I see no reason for using their API currently. I think it's too early for the partnership and I suggest getting back to this when widgets API is available.

Their partnership offer, i.e "Joint promotion of one another's projects is a given as well!" sounds interesting. Do we know how much traffic can they provide?

**Reply by ipaleka** [reply]
> A fair partnership offer here is asastats.com introduces a widget for their service (so users interested in the service can pay for it) and they provide us with their API.

After Davis introduced us to their project, this is exactly what we proposed and pointed out during the meeting as our primary interest for partnership.

**Reply by ipaleka** [reply]
> Their partnership offer, i.e "Joint promotion of one another's projects is a given as well!" sounds interesting. Do we know how much traffic can they provide?

They don't offer their services yet, so I guess they and we can only speculate about that.

**Comment by Motuwagon** [comment]
I agree with the others points of: does the team need the 4 services?
  I wanted to point out it reads like a one year service contract.  ASA pays 6000usd equivalent for one year of services. Included in the year subscription shared marketing.

**Reply by ipaleka** [reply]
Yes, that's basically that.

We are interested in that first point, but as DragMZ pointed out that won't take place until our user widgets are available.

**Comment by Zamanikings** [comment]
I think a partnership would be amazing..... Even though I I'm not sure how the airdrop automation would benefit ASA Stats (I don't think there is an airdrop or plan to be one) so I would suggest a draft be made and see how this partnership benefits us at ASA Stats before we can make a considerable decision regarding this

**Reply by ipaleka** [reply]
Sorry for editing your comment, but we may use whichever name comes to our mind (of course we prefer ASA Stats, but every other example is fine like ASAstats, asastats, asa stats, ASAstats.com, asastats.com, www.asastats.com, ...) for the project **except** for ASASTATS as we are everything but a *token project*.

**Comment by ipaleka** [comment]
A follow-up from Davis:

- - - - -

> Hi Ivica,

> I wanted to follow-up and let you know that we are planning to roll out a non-enterprise plan to use our software. Same functionality, different support structure and a few other bells and whistles. All that to say, if you want to start messing around with our platform. I can go ahead and get you onboarded for 200 USD/ Month (or 2000 USD / year). Let me know if that is of interest to you,

> Davis

- - - - -

I'm going to reply in the next two days with the excerpts from this discussion. Any thoughts?

**Comment by ipaleka** [comment]
- - - - -

Hi Davis!

We had conducted a discussion about your proposal (https://github.com/asastats/channel/discussions/11) and its main points are that we can have use of your future API and that, unfortunately, your proposal isn't acceptable for us.

The first aspect of your proposal is the allocation of tokens, but we have invested in developing an in-house solution for the purpose and we are more than satisfied with it.

The other aspect is checking and surveillance of airdrop farmers, but the matter of fact is that we even don't have any institute and mechanism for punishment in ASA Stats! Our tokens are allocated based on very strict conditions (being qualified for airdrops, rewards for contributing to the project, ...) and once they leave our pools the receivers can do whatever they want with them.

Nevertheless, we are indeed interested in your project, but completely from our users' perspective, and not directly from ASA Stats'. Once your API is published we'll be sponsoring the development of ASA Stats User Widgets (point 16. in our roadmap https://github.com/asastats/docs/blob/main/roadmap.md) which uses your API for delivering your content to our users' pages. Once your API is developed we'll have a discussion should we purchase your API by ourselves or we'll leave that part to the actual third-party developers of BlockRank-based ASA Stats User Widgets.

Sincerely,

Ivica

- - - - -
